rest-server/core/src/main/scala/com/actionml/core/store/backends/MongoConfig.scala

```diff
@@ -17,16 +17,14 @@
 
 package com.actionml.core.store.backends
 
-import com.typesafe.config.ConfigFactory
-import net.ceedubs.ficus.Ficus._
-import net.ceedubs.ficus.readers.ArbitraryTypeReader._
-import net.ceedubs.ficus.readers.namemappers.implicits.hyphenCase
+import com.mongodb.MongoClientURI
 
 
-case class MongoConfig(host: String, port: Int)
+case class MongoConfig(private val uriString: String) {
+  val uri = new MongoClientURI(uriString)
+}
 
 object MongoConfig {
-  private lazy val config = ConfigFactory.load
 
-  lazy val mongo = config.as[MongoConfig]("mongo")
-}
+  val mongo = MongoConfig(sys.env.getOrElse("MONGO_URI", throw new RuntimeException("Environment variable MONGO_URI must be set")))
```

Comment:
@pferrel am I right here?

Reply:
Looks correct, as long as we always set the value in script or code. If this exception is thrown it means someone has removed the setting.